src/main/drivers/vtx_common.c

```diff
@@ -133,6 +142,17 @@ bool vtxCommonGetPitMode(uint8_t *pOnOff)
         return false;
 }
 
+bool vtxCommonGetFrequency(uint16_t *pFreq)
+{
+    if (!vtxDevice)
```

Comment:
This "protection" check is unnecessary as `vtxCommonGetFrequency` is called under protection of `vtxCommonGetDeviceType`.

Not also I generally dislike using protection code like this - better to ensure the caller only calls when appropriate.